### PR TITLE
Eric datatable edit

### PR DESCRIPTION
**Pull request by ericrsmith35**
Implement Cancel/Save Buttons and a couple other minor updates

### CODE REVIEW

**Comment by spider-bkv** [comment]
@ericrsmith35 Let's add an additional attribute to manage visibility of Save/Cancel buttons?

If you don't mind, I can add this tomorrow.

Thank you!

**Comment by ericrsmith35** [comment]
Sounds good. Probably best to build out 2  datatables in the cmp file and
use conditional visibility.

**Comment by ericrsmith35** [comment]
@spider-bkv I went ahead and made the change to have the buttons selectable via attribute.

File: flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp [hunk 1]
```diff
@@ -15,15 +18,17 @@
...
 
 -->
 
 <aura:component implements="lightning:availableForFlowScreens">
 
-	<aura:attribute name="obj" type="String" />
-	<aura:attribute name="mydata" type="SObject[]"/>
+    <aura:attribute name="obj" type="String" />
+    <aura:attribute name="mydata" type="SObject[]"/>
+    <aura:attribute name="budata" type="SObject[]"/>
```
Comment (alexed1) [inline flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp]: perhaps a more evocative attribute name? This is a draft of the data that includes inline changes?  let's add a comment too

File: flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp [hunk 2]
```diff
@@ -135,20 +140,37 @@
     <aura:attribute name="hideShow" type="String" default="show"/>
 
     <aura:attribute name="keyField" type="String" default="Id"/>
+    <aura:attribute name="buttons" type="Boolean" default="false"/>
```
Comment (alexed1) [inline flow_screen_components/datatableFSC/force-app/main/default/aura/datatableFSC/datatableFSC.cmp]: A good practice with boolean attributes is to name them to try and make it clear what the states are. for example isActivated is a better boolean name than status. It's not quite clear to me as a reader what this boolean represents: showButtons? useButtons?

**Comment by ericrsmith35** [comment]
See Pull Request #212 with the suggested updates.

**Pull request merged**